plugins/pulp_ostree/plugins/db/model.py

```diff
@@ -37,6 +37,21 @@ def to_mongo(self, value):
         """
         return dict([(k.replace('.', '-'), v) for k, v in value.items()])
 
+    def validate(self, value):
```

Comment:
I suspect this has been broken for a while, perhaps due to a change in ostree. Saving a model always failed validation, because a dictionary key produced by the ostree library had a '.' in it.